homework_1/zad1/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"zad1/solution"
```

Comment:
Once you rename your module in go.mod (as suggested in a different comment), you will need to change this import path as well.